static/css/style.css

```diff
@@ -586,7 +547,7 @@ select {
 .post__content h1, .post__content h2, .post__content h3, .post__content h4,
 .post__content h5, .post__content h6, .post__content p,
 .post__content blockquote, .post__content .row,
-.post__content .mr-video-container {
```

Comment:
No, I should remove both. 

```
// This will modify only the .mr-video-container that is inside .post__content
.post__content .mr-video-container
```

If I remove just the `.mr-video-container` and let the `.post__content` on this line I will change the attibutes of `.post__content` that is not what I want, I want to let the `.post__content` like it is, without changes.